frame/contracts/src/wasm/runtime.rs

```diff
@@ -1254,6 +1260,32 @@ define_env!(Env, <E: Ext>,
 		)?)
 	},
 
+	// Checks whether a specified address belongs to a contract.
+	//
+	// # Parameters
+	//
+	// - account_ptr: a pointer to the address of the beneficiary account
+	//   Should be decodable as an `T::AccountId`. Traps otherwise.
+	//
+	// Returned value is an u32-encoded boolean: (0 = false, 1 = true)
```

Comment:
`u32` is no vowel sound. Some english quirk.

```suggestion
	// Returned value is a u32-encoded boolean: (0 = false, 1 = true).
```